metricbeat/include/list.go

```diff
@@ -8,6 +8,22 @@ package include
 
 import (
 	// This list is automatically generated by `make imports`
+	_ "github.com/elastic/beats/metricbeat/module/system"
```

Comment:
Was this generated automatically with `make collect` / `make imports? I would expect it to be sorted alphabetically.